llvm/lib/Target/AArch64/AArch64TargetTransformInfo.cpp

```diff
@@ -4703,22 +4769,24 @@ void AArch64TTIImpl::getUnrollingPreferences(Loop *L, ScalarEvolution &SE,
     break;
   }
 
+  if (!L->getExitBlock() && shouldUnrollMultiExitLoop(L, SE, *this)) {
```

Comment:
Could you add a comment explaining what loops we try to target here?

Reply:
Yep, I've tried to add something.